### ISSUE TITLE
Ensure cache is cleared up when user unsubscribes to job

### ISSUE THREAD

**Issue by githubbob42** [githubbob42/mingle2github2#1743]
**[Mingle Card: 1962](https://liquidframeworks.mingle.thoughtworks.com/projects/alpine_mobile/cards/1962)**
Cache invalidation. 

When a user no longer is subscribed to a job, mobile needs to continue to request any urls that were included on the previous sync. When the host page (the points to the manifest) loads, the manifest will respond with a 404 or 410 response which will tell the browser that the cache entry can be removed\* - the cache will persist until the page is refreshed (as per any other manifest update, changes only take on the NEXT request).

tl;dr

We need to listen for the applicationCache.OBSOLETE event, and trigger a fresh to the iframe hosting that manifest file, otherwise the cache will not be cleanup up.

\* see http://www.html5rocks.com/en/tutorials/appcache/beginner/#toc-manifest-file-structure for further information

**Comment by githubbob42** [comment]
### Story: #1671 Mobile should retrieve attachments from Console ([Mingle](https://liquidframeworks.mingle.thoughtworks.com/projects/alpine_mobile/cards/1887))
